swarm/network/stream/snapshot_retrieval_test.go

```diff
@@ -66,6 +68,44 @@ func TestFileRetrieval(t *testing.T) {
 	}
 }
 
+// TestPureRetrieval tests pure retrieval without syncing
+// A configurable number of nodes and chunks
+// can be provided to the test.
+// A number of random chunks is generated, then stored directly in
+// each node's localstore according to their address.
+// Each chunk is supposed to end up at certain nodes
+// With retrieval we then make sure that every node can actually retrieve
+// the chunks.
+func TestPureRetrieval(t *testing.T) {
+	var nodeCount []int
+	var chunkCount []int
+
+	if *nodes != 0 && *chunks != 0 {
+		nodeCount = []int{*nodes}
+		chunkCount = []int{*chunks}
+	} else {
+		nodeCount = []int{16}
+		chunkCount = []int{16}
```

Comment:
This is a rather very low number of chunks. If we are going to be starting 16 nodes, we might as well load/store a few more chunks, specially when they are generated randomly.

Reply:
Obligatory joke
```
A devops engineer walks into a bar, puts the bartender in a docker container, put kubernetes behind the bar, spins up 1000 bartenders, orders 1 beer.
```